Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs

```diff
@@ -66,6 +69,7 @@ public LandscapePlugin(IRealmData realmData,
             this.worldTerrainGenerator = worldTerrainGenerator;
 
             parcelService = new LandscapeParcelService(webRequestController, isZone);
+            wrapper = GPUIWrapperFactory.CreateGPUIWrapper();
```

Comment:
So instead of doing an interface and a fake implementation, I would have instead wrapped every use in #if. Or just leave the `wrapper` null and use `?.`.

Reply:
Agreed. Removing the boilerplate